apcd-cms/src/apps/admin_regis_table/views.py

```diff
@@ -56,11 +56,22 @@ def _set_entities(reg_ent):
                 'fein': reg_ent[8]
             }
         def _set_contacts(reg_cont):
+
+            def format_phone_number(num):
+                formatted_num_list = list(reversed(num))
+                for n in range(len(formatted_num_list)):
+                    if n+1 == 4 or n+1 == 8:
```

Comment:
Would this be more readable? If not, feel free to ignore me
```suggestion
                    if n == 3 or n == 7:
```